source/Posets/2ElementPoset.lagda

```diff
@@ -0,0 +1,51 @@
+Ian Ray, 9 August 2023
```

Comment:
Maybe rename the file to `TwoElementPoset`.

Reply:
When I changed the file name it created a duplicate and now both are in the PR. Can I delete the original 2ElementPoset from the PR here on GitHub or do I need to do something locally.

Reply:
To get renamings to be registered by `git`, you typically do `git mv old new`. Maybe you can delete the old file from the PR by doing `git rm 2ElementPoset.lagda`. Make sure you have some backups before you do anything drastic :slightly_smiling_face: 

We can always delete the old file later in some PR, so don't worry too much about it.